### PR TITLE
read heartbeat token from operator contract

### PR DESCRIPTION
**Pull request by PumpkingWok**
Please remember the [Contributing Guidelines](https://github.com/leapdao/meta/blob/master/CONTRIBUTION.md) :heart:

## What does this PR do?

- [X] Read heartbeat token from operator contract.
- [X] Update unit tests.

## Recommendations for testing

Test both `leap-node` and the network using `leap-sandbox`.

## Links to relevant issues or information

Fix: #413

### CODE REVIEW

File: src/heartbeat/index.js [hunk 1]
```diff
@@ -25,9 +25,12 @@ function updateConfig({ heartbeat = {} }) {
   return { ...defaults, ...heartbeat };
 }
 
-module.exports = (bridgeState, sender) => {
+module.exports = async (bridgeState, sender) => {
```
Comment (troggy) [inline src/heartbeat/index.js]: since you made this function async, it's better to call it with `await` now

**Comment by PumpkingWok** [comment]
Hi @troggy,
I have just commited some fixes, i noticed that maybe in `heartbeat` `index.test.js` there are 2 tests that are not useful anymore, i think.thanks

File: src/heartbeat/pulse.js [hunk 2]
```diff
@@ -3,10 +3,13 @@ const { Tx, Input, Outpoint, Output } = require('leap-core');
 const getUnspent = require('../api/methods/getUnspent');
 
 module.exports = async (bridgeState, { send }) => {
+  const heartbeatColor = await bridgeState.operatorContract.methods
```
Comment (troggy) [inline src/heartbeat/pulse.js]: pulse is being called by setTimeout many times. indefinitely. No reason to call heartbeatColor() on the contract everytime — it won't change often (if ever)

**Comment by troggy** [comment]
Is it ready for review?

**Comment by PumpkingWok** [comment]
Sure, thanks a lot.

**Pull request merged**